### ISSUE TITLE
Wishlist: Need a mandatory sunset clause

### ISSUE THREAD

**Issue by anivar** [nic-delhi/AarogyaSetu_Android#3]
All proximity tracing applications take too many critical device permissions, and it should sunset from functioning after the covid19 crisis period. 

A sunset policy is essential for all proximity tracing apps.

**Comment by Thedijje** [comment]
You are right, but this is yet not decided and can not be added as of now, must be planned for later on. Someone should code the same with some expected date.

**Comment by FarmaanElahi** [comment]
It would be good if somebody can specify what data is being captured currently, so that anybody seeing this issue will have decent idea, what need to be done resolve this issue

**Comment by damooo** [comment]
@anivar  here is [list](https://www.cvedetails.com/vulnerability-list.php?vendor_id=1224&product_id=19997&version_id=0&page=1&hasexp=0&opdos=0&opec=0&opov=0&opcsrf=0&opgpriv=0&opsqli=0&opxss=0&opdirt=0&opmemc=0&ophttprs=0&opbyp=0&opfileinc=0&opginf=0&cvssscoremin=0&cvssscoremax=0&year=0&cweid=0&order=1&trc=2563&sha=1bd76566e804bd0baf4aa6ef43598ed24565b5b6) of thousands of known cves in android as platform.

Should we list one by one on start screen to every ordinary user, and ask confirmation too?

**Comment by damooo** [comment]
@anivar  here is [list](https://www.cvedetails.com/vulnerability-list.php?vendor_id=1224&product_id=19997&version_id=0&page=1&hasexp=0&opdos=0&opec=0&opov=0&opcsrf=0&opgpriv=0&opsqli=0&opxss=0&opdirt=0&opmemc=0&ophttprs=0&opbyp=0&opfileinc=0&opginf=0&cvssscoremin=0&cvssscoremax=0&year=0&cweid=0&order=1&trc=2563&sha=1bd76566e804bd0baf4aa6ef43598ed24565b5b6) of thousands of known cves in android as platform.

Should we list one by one on start screen to every ordinary user, and ask confirmation too?

foss activism doen't mean jingoism, and hysteria. It should consider practicality, value developers, should not encumber ordinary people.
